admin/c2cgeoportal_admin/__init__.py

```diff
@@ -1,13 +1,24 @@
+import c2cwsgiutils.pyramid
+# from c2cgeoportal_commons import models
```

Comment:
Why added a commented import?

Reply:
This will be useful in the future, when the line below with `health_check.add_db_session_check` will be uncommented

Reply:
ok